src/components/Checkbox/Checkbox.scss

```diff
@@ -76,11 +76,11 @@
         @include control-backdrop(active, $global-theming: true);
       }
       ~ .Icon {
+        transition: opacity var(--p-duration-1-5-0) var(--p-ease),
+          transform var(--p-duration-1-5-0) var(--p-ease);
         transition-timing-function: var(--p-ease-in);
```

Comment:
This will now likely override the timing function specified above (not sure why it's been kept there in the first place)

Reply:
@BPScott I just saw the change in 0da64f2f43a1acf452246e737930cea2e72b6f08 and I don't think this is the right rewrite, as the cascade in the original version would have ignored `transition-timing-function: var(--p-ease-in);`.

This should work:

```diff
+        transition: opacity var(--p-duration-1-5-0) var(--p-ease),
+          transform var(--p-duration-1-5-0) var(--p-ease);
-        transition-timing-function: var(--p-ease-in);
-        transition: opacity var(--p-duration-1-5-0) var(--p-ease),
-          transform var(--p-duration-1-5-0) var(--p-ease);
```

Reply:
done!